### ISSUE TITLE
Alternative checkboxes support

### ISSUE THREAD

**Issue by approximate** [dracula/obsidian#30]
Hi, and thanks for supporting this great theme in Obsidian!

Are there any plans to implement alternative checkboxes (like the [ITS Theme](https://github.com/SlRvb/Obsidian--ITS-Theme/blob/main/Guide/Alternate-Checkboxes.md), among several others, does)?

I personally don't use all of these, but for my task management workflow supporting at least "cancelled" and "in progress" tasks would be a big help.

Thanks again!

**Comment by Chrismettal** [comment]
Good morning!

There are currently no plans for me to implement this, as it diverges from the Markdown standard, which I personally just don't use, as Obsidian isn't the only software I use to access my knowledge base.

I am by no means opposed to having this feature in there, I just won't make the time to add a feature I personally can't use. I would happily merge this feature if you open up a PR for it!

**Comment by electblake** [comment]
tasks are a critical workflow in my obsidian vault, especially have learned to love the extended task statuses (incomplete, scheduled etc) - I found it actually easier to utilize [minimal theme](https://github.com/kepano/obsidian-minimal), [style settings plugin](https://github.com/mgmeyers/obsidian-style-settings) and then using the [dracula colors palette from wikipedia](https://en.wikipedia.org/wiki/Dracula_(color_scheme)#Color_palette).

Didn't take long to get going, and gives me best of both worlds

Note, I based my initial changes on the [screenshot provided](https://github.com/dracula/obsidian/blob/main/screenshot_1080p.png) by this theme (but it isn't a full kitchen sink of all obsidian features so I'll likely need to do theme switching to get a better mirror of this theme in Minimal)

If it's supported by the draculatheme project, I'll create repo and upload my settings for community (Minimal Theme / Style Settings has an import/export function which makes it easy to share!)

**Comment by Chrismettal** [comment]
I personally would much rather merge an alternate checkbox implementation into this official Dracula repo, than opening up a second repo providing similar but non-identical features, but sharing your settings doesn't hurt!

**Comment by electblake** [comment]
Yeah, I can appreciate that for sure - but if this theme won't have updates to support Obsidian features (vs. following markdown standards) - I doubt I'll invest more into this theme :(

Minimal (and other popular themes) support new features as they come out, such as the latest core plugin for Obsidian Properties, Canvas or even community plugins like dataview (or tasks, which is what provides a lot of the tooling for these extended task statuses).

Personally, I'm looking for a full-featured obsidian theme that looks beautiful and tailored in all of the features I use - so maybe I've priced myself out of this theme 🤷🏼 

Admittedly, keeping up with these features must be a pain, and I have no interest in mining and maintaining css selectors, so I'd rather piggyback on a theme who does.

Really enjoy the draculatheme project, and I hope to find another avenue to contribute nonetheless 🙏🏼
